src/index.js

```diff
@@ -1,2 +1,5 @@
-export { default as Footer, EVENT_NAMES } from './components/Footer';
-export { default as messages } from './i18n/index';
+import Footer, { EVENT_NAMES } from './components/Footer';
+import messages from './i18n/index';
+
+export { Footer, messages, EVENT_NAMES };
```

Comment:
nit: While there's no harm in keeping this named export here (and we probably should to avoid needing to release this as another major breaking change), we may want to add a comment as a reminder to our future-selves on why we're keeping this named `Footer` export here.

Reply:
because move of version 5 release has that. I would rather keep it just make sure it is consistence to other v5 release.

Reply:
Right, I understand that. I'm just suggesting we add a comment to explain why we're keeping the named `Footer` export _and_ the default `Footer` export as a helpful reminder for us in the future (so when we look at this again in a couple months, we don't get confused why we're exporting `Footer` twice.

Reply:
A comment in the code itself, not just a PR comment.